docs/pages/preview/upcoming-releases.mdx

```diff
@@ -40,24 +40,24 @@ You can find the full list of fixes and features in the [Github milestone](https
 | ------- | ----------------- | ------------------- |
 | Release | August 30th, 2021 | Production release. |
 
-## Teleport Cloud "Washington"
+## Teleport Cloud
 
-The Cloud team improves user experience - adds databases and kubernetes user interfaces
-and wizards, adds downloads page.
-The team improves cloud self service with password recovery and cancellation, adds data export.
+The early 2022 focus for the Teleport Cloud product is to focus on 3 key areas:
```

Comment:
I think we can tighten up this sentence to something like, "The key areas of focus for Teleport cloud in early 2022 are:"